### PR TITLE
WIP: tutorials from export, renamed for schedule.

### PR DESCRIPTION
**Pull request by FlipperPA**

### CODE REVIEW

**Comment by FlipperPA** [comment]
Just a note: please don't merge this quite yet (although that green button looks SO shiny!). We're giving tutorial presenters a final chance to update their info in PaperCall, so there may be a few touch ups. THANKS!

File: _schedule/tutorials/2018-10-14-09-00-t2-python-packaging-without-the-pain.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
+---
+abstract: Many of us have snippets of Python code we copy from project to project. Wouldn't it be nice if we could `pip install` them from PyPI instead of copy-pasting everywhere? But packaging has always been a complicated, scary task. We will take you from copy/paste to `pip install` in one tutorial!
+accepted: true
+category: tutorial
+date: 2016-07-18 09:00
+difficulty: Intermediate
+layout: session-details
+permalink: /tutorial/python-packaging-without-the-pain/
+presenters:
+- bio: "Diane Chen has 20+ years of experience programming Engineering CAD/CAM software, has built a few websites and worked in hardware testing.\r\nEnamored with the Python programming language, she teaches Python at [UC San Diego Extension](http://bit.ly/UCSDPython), workshops and private courses.
```
Comment (kennethlove) [inline _schedule/tutorials/2018-10-14-09-00-t2-python-packaging-without-the-pain.md]: Will that `\r\n` mess with anything?

Comment (KatherineMichel) [reply _schedule/tutorials/2018-10-14-09-00-t2-python-packaging-without-the-pain.md]: Not sure what it's intended to do.
screen shot 2018-07-20 at 2 22 13 pm

**Comment by jefftriplett** [comment]
@FlipperPA I can help with formatting, but @cholmes5 noticed that all six start at 9am ;)

File: _schedule/tutorials/2018-10-14-09-00-t2-python-packaging-without-the-pain.md (hunk 1 above)
Comment (FlipperPA) [reply _schedule/tutorials/2018-10-14-09-00-t2-python-packaging-without-the-pain.md]: Good eye! I caught most of them...

**Pull request merged**